modules/service/src/main/scala/lucuma/odb/graphql/OdbMapping.scala

```diff
@@ -222,7 +230,11 @@ object OdbMapping {
               ).combineAll
             )
 
-          // Override `fetch` to log the query. This is optional.
+          // Override `defaultRootCursor` to log the GraphQL query. This is optional.
+          override def defaultRootCursor(query: Query, tpe: Type, env: Env): F[Result[(Query, Cursor)]] =
+            Logger[F].info("\n\n" + PrettyPrinter.query(query).render(100) + "\n") *> super.defaultRootCursor(query, tpe, env)
```

Comment:
As with the override of `fetch` below this just prints stuff out and you can comment it out if it bothers you.